ganttproject/src/net/sourceforge/ganttproject/search/SearchResult.java

```diff
@@ -19,28 +19,51 @@
 package net.sourceforge.ganttproject.search;
 
 public class SearchResult<SearchObjectType> {
+  private final int id;
   private final String myOrigin;
-  private final String mySnippet;
-  private final String myLabel;
+  private final String secondaryLabel;
+  private final String typeOfResult;
+  private final String secondaryText;
+  private final String myLabel; //Top line
+  private final String myQueryMatch;
   private final SearchService<?, SearchObjectType> mySearchService;
   private final SearchObjectType mySearchObject;
 
-  public SearchResult(String label, String snippet, String origin, SearchObjectType searchObject,
-      SearchService<?, SearchObjectType> service) {
+  public SearchResult(
+          int id,
```

Comment:
Also style comment: we use two-space indents and four-space line continuation.

Reply:
That's what I get for using netbeans instead of IntelliJ. Fixing.